server/data_tiledb/tiledb_engine.py

```diff
@@ -0,0 +1,288 @@
+import os
+import json
+from server.data_common.driver import CXGDriver
+import tiledb
+import numpy as np
+import pandas as pd
+from server.data_common.fbs.matrix import encode_matrix_fbs
+from server.common.utils import path_join
+from server_timing import Timing as ServerTiming
+from server.common.constants import Axis
+
+
+class TileDbEngine(CXGDriver):
+    def __init__(self, location=None, config=None):
+        super().__init__(config)
+        self.url = location
+        self.update(location, config)
+        self.tiledb_ctx = tiledb.Ctx({
+            'sm.tile_cache_size': 8 * 1024 * 1024 * 1024,
```

Comment:
Ideally this would be configurable from the environment.  Suggest adding a comment to that effect so we remember later. No need to hook it up today.

Reply:
I added a comment.